src/fontra/views/editor/scene-controller.js

```diff
@@ -504,6 +504,41 @@ export class SceneController {
     return undoInfo !== undefined;
   }
 
+  async doDelete() {
+    await this.editInstance((sendIncrementalChange, instance) => {
+      const { point: pointSelection } = parseSelection(this.selection);
+
+      const changes = recordChanges(instance, (instance) => {
+        const path = instance.path;
+
+        let offset = 0;
+        for (const point of pointSelection) {
+          const [contourIndex, contourPointIndex] = path.getContourAndPointIndex(point);
+          const selectedContours = getSelectedContours(path, pointSelection);
+          const contour = path.getUnpackedContour(contourIndex);
```

Comment:
To get at the number of points in a contour, you can do 
```js
const numContourPoints = path.getNumPointsOfContour(contourIndex);
```
This allows you to avoid the `getUnpackedContour()` call.